processing-service-container/src/test/java/com/github/cafdataprocessing/processing/service/tests/GlobalConfigIT.java

```diff
@@ -79,19 +78,29 @@ public void createGlobalConfigWithKeyLongerThan255() throws ApiException
         }
     }
     
+    @Test
+    public void createGlobalConfigWithoutScopeParameter() throws ApiException 
+    {
+        try {
+            globalConfigAPI.setGlobalConfig(TEST_KEY_PREFIX, buildGlobalConfig(TEST_VALUE_1, TEST_DESCRIPTION_1, null));
+        } catch (ApiException e) {
+            assertTrue(e.getCode() == 400);
```

Comment:
Use assertEquals here as it'll give a clearer message in the log e.g. if this is false assertTrue just output that the condition was false when true was expected but assertEquals will output both values if they don't match.